core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -638,12 +640,16 @@ Builder populate() {
       defineMethod(EXP, BuiltInMethod.EXP.method, NullPolicy.STRICT);
       defineMethod(POWER, BuiltInMethod.POWER.method, NullPolicy.STRICT);
       defineMethod(ABS, BuiltInMethod.ABS.method, NullPolicy.STRICT);
-      defineMethod(LOG2, BuiltInMethod.LOG2.method, NullPolicy.STRICT);
 
       map.put(LN, new LogImplementor());
       map.put(LOG, new LogImplementor());
       map.put(LOG10, new LogImplementor());
 
+      map.put(LN, new LogMSImplementor());
```

Comment:
map is a Java Map, you are overwriting the previous entry.

Reply:
ohhh, thank you for reminding me, I made a serious mistake